### PR TITLE
Bug 1133729 - Migrate test-fxa-client app from dev_apps to fixtures folder

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1133729

### CODE REVIEW

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0157caea944a)

File: apps/system/fxa/test/marionette/fxa_launch_test.js [hunk 1]
```diff
@@ -20,6 +20,9 @@ marionette('Firefox Accounts Launch Tests', function() {
           },
           settings: {
             'lockscreen.enabled': false
+          },
+          apps: {
+            'test-fxa-client.gaiamobile.org': + __dirname + '/test-fxa-client',
```
Comment (borjasalguero) [inline apps/system/fxa/test/marionette/fxa_launch_test.js]: Should we add `/* global __dirname */` at the top of the file?

Comment (KevinGrandon) [inline apps/system/fxa/test/marionette/fxa_launch_test.js]: I don't think this is necessary as linters are currently passing. The system .jshintrc file already defines the environment as node: https://github.com/mozilla-b2g/gaia/blob/master/apps/system/test/marionette/.jshintrc#L3

(This is something we should probably do everywhere)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=670575d3e4b5)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=95f1f187eb37)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d3bcf3506553)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9de7172f9d4d)

**Pull request merged**